manifests/0000_50_olm_00-namespace.yaml

```diff
@@ -16,8 +18,6 @@ kind: Namespace
 metadata:
   name: openshift-operators
   labels:
-    pod-security.kubernetes.io/enforce: baseline
-    pod-security.kubernetes.io/enforce-version: "v1.24"
```

Comment:
until we have the OLM logic in place to label "openshift-* namespaces that have operators installed" for labelsyncing, we should probably leave this explicit setting in place (or even set it to privileged), so that operators that are installed in this NS don't get rejected by PSA.

cc @perdasilva

Reply:
I think needs to be previlged we do not know what will run in openshift-operators. It is the default namespace to install the projects.

Reply:
That is a good point. I've modified the PR to go with option 1 and explicitly label it as `privileged`, so that when https://issues.redhat.com/browse/OLM-2695 is ready we have to make a conscious effort to remove any hard-coding of labels in this namespace.